### PR TITLE
Update the release note of 2021.1

### PR DESCRIPTION
**Pull request by saugatt**
Update the gh-pages-2021.1 with what's new for 2021.1

### CODE REVIEW

File: _posts/2021-03-29-new-release-2021-1.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title:  "Tableau Connector SDK for 2020.4 Released, latest version of SDK now targets 2021.1"
```
Comment (rosswbrown) [inline _posts/2021-03-29-new-release-2021-1.md:2]: Try adjusting the working in the title and abstract to more closely match the [others](https://tableau.github.io/connector-plugin-sdk/news/).  It'll make it more clear we're releasing 2021.1.

Comment (saugatt) [reply _posts/2021-03-29-new-release-2021-1.md]: Updated the title to reflect the wording in title to match other.

File: _posts/2021-03-29-new-release-2021-1.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+title:  "Connector SDK now targets Tableau 2021.1"
+abstract: "The master SDK branch now targets Tableau 2021.1. The Connector SDK for 2020.4 has released for those wishing to target that version."
+---
+
+On March 29th, with the full release of Tableau 2021.1 we moved the latest version of the master branch to target Tableau 2020.1. This means that the samples and connectors made with this version of the SDK may not work with older versions of Tableau. To develop against Tableau 2020.4, please download the [Tableau Connector SDK for Tableau 2020.4](https://github.com/tableau/connector-plugin-sdk/releases/tag/tableau-2020.4) release.
+
+- __OAuth Authentication Support__ <br/>
+ Starting in Tableau 2021.1 a connector using Connection Dialog V2 style, OAuth is supported as one of the categories for authetication. To learn more please refer  [here]({{ site.baseurl }}/docs/oauth). For a complete example please refer to [here](https://github.com/tableau/connector-plugin-sdk/tree/dev-2021.1/samples/scenarios/snowflake_oauth).
```
Comment (rosswbrown) [inline _posts/2021-03-29-new-release-2021-1.md:9]: Update link to use master now that it is 21.1

File: _posts/2021-03-29-new-release-2021-1.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+- __Runtime Determination of Connection Normalizer__ <br/>
+Starting in Tableau 2021.1 a connector using Connection Dialog V2 style, connection-fields, can let the system determine at runtime the correct connection-normalizer by not defining it. Any connection-normalizer defined in the tdr file will take precedence at runtime. To use set `min-version-tableau='2021`.1' or newer in the manifest file.
```
Comment (rosswbrown) [inline _posts/2021-03-29-new-release-2021-1.md:11]:
```suggestion
Starting in Tableau 2021.1 a connector using Connection Dialog V2 style, connection-fields, can let the system determine at runtime the correct connection-normalizer by not defining it. Any connection-normalizer defined in the tdr file will take precedence at runtime. To use set `min-version-tableau='2021.1'` or newer in the manifest file.
```

File: _posts/2021-03-29-new-release-2021-1.md (hunk 2 above)
Comment (saugatt) [reply _posts/2021-03-29-new-release-2021-1.md]: Updated the link to reflect master branch

**Review by rosswbrown — APPROVED** [review]

**Pull request merged**